library/src/main/java/com/bumptech/glide/request/RequestListener.java

```diff
@@ -68,18 +69,19 @@ boolean onLoadFailed(
    *
    * <p>For threading guarantees, see the class comment.
    *
-   * @param resource The resource that was loaded for the target.
-   * @param model The specific model that was used to load the image.
-   * @param target The target the model was loaded into.
-   * @param dataSource The {@link DataSource} the resource was loaded from.
+   * @param resource        The resource that was loaded for the target.
+   * @param model           The specific model that was used to load the image.
+   * @param target          The target the model was loaded into.
+   * @param dataSource      The {@link DataSource} the resource was loaded from.
    * @param isFirstResource {@code true} if this is the first resource to in this load to be loaded
-   *     into the target. For example when loading a thumbnail and a full-sized image, this will be
-   *     {@code true} for the first image to load and {@code false} for the second.
+   *                        into the target. For example when loading a thumbnail and a full-sized image, this will be
+   *                        {@code true} for the first image to load and {@code false} for the second.
    * @return {@code true} to prevent {@link Target#onResourceReady(Object, Transition)} from being
-   *     called on {@code target}, typically because the listener wants to update the {@code target}
-   *     or the object the {@code target} wraps itself or {@code false} to allow {@link
-   *     Target#onResourceReady(Object, Transition)} to be called on {@code target}.
+   * called on {@code target}, typically because the listener wants to update the {@code target}
+   * or the object the {@code target} wraps itself or {@code false} to allow {@link
+   * Target#onResourceReady(Object, Transition)} to be called on {@code target}.
```

Comment:
Same here, please revert the javadoc spacing changes

Reply:
Not mine. I don't know what to do. Only change I had is to add the annotation:

![image](https://user-images.githubusercontent.com/5357526/190845604-dd61ccae-51d3-430f-93b2-2cafcd832553.png)